src/main/java/duke/DukeStub.java

```diff
@@ -0,0 +1,10 @@
+package duke;
+
+/**
+ * Stub for a completed Duke that can respond to user input.
+ */
+public class DukeStub {
```

Comment:
If the stub was created for testing purpose, you may want to move it to the `test` folder